src/Robo/Inspector/Inspector.php

```diff
@@ -287,9 +287,12 @@ public function isDrupalVmConfigPresent() {
   public function isDrupalVmLocallyInitialized() {
     $status = $this->getDrupalVmStatus();
     $machine_name = $this->getConfigValue('project.machine_name');
+    $local_project_config = $this->getLocalProjectVmConfig();
+    $local_drush_alias = $local_project_config->get('drush.aliases.local');
```

Comment:
It shouldn't matter which config file this configuration lives in. It's valid to define this in project.yml, have _no_ project.local.yml, and still expect a VM. Instead, I think we need separate variable like `vm.enable` that more directly represents isDrupalVmLocallyInitialized().